testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeSqlserver.java

```diff
@@ -56,7 +56,7 @@ public void extendEnvironment(Environment.Builder builder)
     @SuppressWarnings("resource")
     private DockerContainer createSqlServer()
     {
-        DockerContainer container = new DockerContainer("mcr.microsoft.com/mssql/server:2017-latest", "sqlserver")
+        DockerContainer container = new DockerContainer("mcr.microsoft.com/mssql/server:2019-CU28-ubuntu-20.04", "sqlserver")
```

Comment:
Was it fixed on 2019 ?

Reply:
Yes, it was fixed in 2019

Reply:
Can we update the latest to 2023 in this case ? Do we need to update the docs for the same ?

Reply:
Let's keep it on 2019, 2023 seems to new ;)

Reply:
I'll update the docs